src/ui/atoms/card/card.scss

```diff
@@ -19,37 +19,36 @@ $sizes-map: (
 
 .okp4-card-main {
   @include with-theme() {
-    clip-path: polygon(30px 0, 100% 0, 100% 100%, 0 100%, 0 30px);
-    padding: 1px;
-
-    &.border {
-      background: themed(border);
-    }
-
-    .okp4-card-container {
-      @each $color-key, $color-value in $colors {
-        display: flex;
-        flex-direction: column;
-        justify-content: space-between;
-        gap: 10px;
-        box-sizing: border-box;
+    @each $size in $sizes {
+      &.#{$size} {
         clip-path: polygon(30px 0, 100% 0, 100% 100%, 0 100%, 0 30px);
-        max-height: map-get($sizes-map, large-height) - 2px;
-        max-width: map-get($sizes-map, large-width) - 2px;
-        overflow: auto;
-        word-break: break-word;
+        padding: 1px;
```

Comment:
I think you could move `clip-path`, `padding` and `width` just above `@include with-theme()` because it is not depending on size.

Reply:
Good point I will refactor the code 😃 👍